wagtailgridder/templates/wagtailgridder/grid_item.html

```diff
@@ -1,51 +1,50 @@
 {% extends "base.html" %}
 
 {% load staticfiles %}
-
 {% load wagtailcore_tags %}
 {% load wagtailimages_tags %}
 
 {% block extra_head_top %}
-<link rel="stylesheet" href="{% static 'wagtailgridder/css/grid_index_page.css' %}">
-<link rel="stylesheet" href="{% static 'wagtailgridder/css/grid_item.css' %}">
+    <link rel="stylesheet" href="{% static 'wagtailgridder/css/grid_index_page.css' %}">
 {% endblock extra_head_top %}
 
 {% block inner_content %}
-    <div class="row">
-        <h2>{{ page.title }}</h2>
-    </div>
-    <div class="row">
-        <div class="col-md-4">
-            {% image page.description_image original %}
+    <div id="wg-grid-item-content">
+        <div class="row">
+            <h2>{{ page.title }}</h2>
         </div>
-        <div class="col-md-8">
-            {{ page.landing_page_text|safe }}
-
-            {% block buttons %}
-                {% for button_section in page.buttons %}
-                    <div class="row{# if forloop.first %} align-items-start{% endif %}{% if forloop.last %} align-items-end{% endif #}">
-                        {% for action_item in button_section.value.action_items %}
-                            <div class="col">
-			        {% block action_item %}
-                                    {{ action_item }}
-				{% endblock action_item %}
-                            </div>
-                        {% endfor %}
-                    </div>
-                {% endfor %}
-            {% endblock buttons %}
+        <div class="row">
+            <div class="col-md-4 text-center pb-2">
+                {% image page.description_image original class="img-fluid center-block" %}
+            </div>
+            <div class="col-md-8 pb-2">
+                {{ page.landing_page_text|safe }}
+                {% block buttons %}
+                    {% for button_section in page.buttons %}
+                        <div class="row">
+                            {% for action_item in button_section.value.action_items %}
+                                <div class="col-md-12 border-bottom border-secondary">
+                                    {% block action_item %}
+                                        {{ action_item }}
+                                    {% endblock action_item %}
+                                </div>
+                            {% endfor %}
+                        </div>
+                    {% endfor %}
+                {% endblock buttons %}
+            </div>
         </div>
     </div>
 {% endblock inner_content %}
 
 {% block extra_footer_js %}
     <script>
-    $(document).ready(function() {
-        for (var i=0; i<$(".default-fa" ).length; i++){
-            if ($(".default-fa" )[i].href.includes("/otis/")) {
-                $(".default-fa" )[i].setAttribute('target', '_blank');
+        $(document).ready(function () {
+            for (var i = 0; i < $(".default-fa").length; i++) {
+                if ($(".default-fa")[i].href.includes("/otis/")) {
```

Comment:
I think we can actually get rid of this, since we no longer have OTIS linked from anywhere! This was a hack to open OTIS in a new window.